.github/workflows/jaws-test.yml

```diff
@@ -0,0 +1,205 @@
+        required: false
+        type: string
+      jaws_url:
+        description: |
+          The download URL for the JAWS installer.
+        required: false
+        type: string
+
+# TODO - JAWS specfic version selection param some day
+
+      # nvda_version:
+      #   description: |
+      #     The specific version of NVDA to use (e.g., '2023.1'). If not provided, the latest version will be used.
+      #   required: false
+      #   type: string
+
+env:
+  ARIA_AT_WORK_DIR: ${{ inputs.work_dir || 'tests/alert' }}
+  ARIA_AT_TEST_PATTERN: ${{ inputs.test_pattern || '{reference/**,test-*-jaws.*}'}}
+  ARIA_AT_CALLBACK_URL: ${{ inputs.callback_url }}
+  ARIA_AT_STATUS_URL: ${{ inputs.status_url }}
+  ARIA_AT_CALLBACK_HEADER: ${{ inputs.callback_header || 'x-header-param:empty' }}
+  BROWSER: ${{ inputs.browser || 'chrome' }}
+  # TODO - this should probably be a version number at some point but we will use this ENV
+  # var in JAWS\InstallJAWSUnattended.ps1
+  JAWS_VERSION: ${{ inputs.jaws_url || 'https://software.vfo.digital/JAWS/2025/2025.2506.168.400/BC70E7A7-B2FA-40DA-A2EC-5595795C65D4/J2025.2506.168.400-enu-x64.exe' }}
+
+jobs:
+  jaws-test:
+    runs-on: windows-2025
+    steps:
+      - name: Log job state QUEUED
+        shell: powershell
+        if: inputs.status_url
+        run: |
+          $headerbits = $env:ARIA_AT_CALLBACK_HEADER -split ":\s*", 2
+          $headers = @{$headerbits[0]=$headerbits[1]; "Content-Type" = "application/json"}
+          $body = @{'status'='QUEUED'; 'externalLogsUrl'="$env:GITHUB_SERVER_URL/$env:GITHUB_REPOSITORY/actions/runs/$env:GITHUB_RUN_ID"} | ConvertTo-JSON
+          Invoke-WebRequest $env:ARIA_AT_STATUS_URL -Headers $headers -Method 'POST' -Body $body
+
+      # Checkout all repos first (helps cache purposes)
+      - uses: actions/checkout@v4
+
+      - name: Checkout aria-at ref ${{ inputs.aria_at_ref || 'master' }}
+        uses: actions/checkout@v4
+        with:
+          repository: "w3c/aria-at"
+          path: "aria-at"
+          ref: ${{ inputs.aria_at_ref || 'master' }}
+
+      # TODO - This version of aria-at-automation-harness currently does not include
+      # JAWS support, but I should have a fix for this soon.
+      - name: Install the ARIA-AT Automation Harness package
+        run: npm install aria-at-automation-harness@"1.0.0"
+
+      - name: Use Node.js 20.x
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
+          cache: "npm"
+          cache-dependency-path: "**/package-lock.json"
+
+      - uses: browser-actions/setup-chrome@v1
+        if: (inputs.browser || 'chrome') == 'chrome'
+        with:
+          chrome-version: stable
+
+      - uses: browser-actions/setup-firefox@v1
+        if: inputs.browser == 'firefox'
+
+      # Installing Scream as a virtual audio driver to avoid exceptions later
+      - name: Install Scream (Virtual Audio Driver)
```

Comment:
I'm curious to hear more about the reasoning for adding this. I don't have any arguments against, just curious!

Reply:
honestly - it was present in the yml suite for very early versions (added by Z) - I'm not 100% sure it's needed, we might want to try an experiment of removing it, I've just been copying it forward for years.